docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -276,4 +281,75 @@ overall values for the data:
     count    0.222222
     dtype: float64
 
-In every case, the *largest* difference and *smallest* ratio are returned.
\ No newline at end of file
+In every case, the *largest* difference and *smallest* ratio are returned.
+
+
+.. _assessment_predefined_fairness_metrics:
+
+Predefined fairness metrics
+---------------------------
+
+In addition to the disaggregated analysis of base metrics enabled by
+:class:`MetricFrame`, Fairlearn also provides a set of predefined fairness 
+metrics that output a single score. These metrics take as input 
+`sensitive_features` to compute the maximum difference or ratio between 
+subgroups of a sensitive variable. The predefined fairness metrics offered 
+by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
+ratio/difference. The ratio and difference can be calculated `between_groups` 
+or `to_overall`, but `to_overall` results in more than 1 number being returned.
```

Comment:
Nitpick: "more than a single value being returned"

But.... when does it do that? I'm confused, because I thought that the `between_groups`/`to_overall` selector just changed the reference point, but that ratio and difference would work as normal.

Reply:
Made the small change from number to value. @romanlutz can you please clarify what you meant above for @riedgar-ms?